app/assets/stylesheets/orders.scss

```diff
@@ -41,9 +47,22 @@
 
     td {
       border-top: 1px dashed $silver_chalice_color;
+
+      .quantity_select {
```

Comment:
Time to be consistent with CSS classes/IDs naming. I suggest using dashes for everything (instead of underscores).